submissions/rubyAdeniregun/Database/Queries.sql

```diff
@@ -0,0 +1,33 @@
+/* Question 1: Query to find the total number of passengers that survived 
+The total number of passengers who survived = 342. */
+SELECT * FROM accident_records WHERE passenger_survived LIKE 1; 
+
+
+/* Question2: Query to find the total number of passengers who did not survive
+The total number of passengers who did not survive = 549. */
+SELECT * FROM accident_records WHERE passenger_survived LIKE 0; 
```

Comment:
The answer is `549` but this doesn't shows that is the query that made you obtain that result.